src/containers/app/index.js

```diff
@@ -13,6 +13,7 @@ const App = () => (
       <Route exact path="/" component={Home} />
```

Comment:
I think Anthony moved routing into the Nav component but ill take care of that if thats the case